lib/ts/recipe/session/types.ts

```diff
@@ -226,6 +249,7 @@ export type RecipeInterface = {
         userId: string;
         accessTokenPayload?: any;
         sessionData?: any;
+        claimsToAdd?: SessionClaimBuilder[];
```

Comment:
We can replace this config with another recipe interface function like `getClaimsToAddOnSessionCreate(userId, defaultClaimsToAddOnCreation, userContext) => Promise<SessionClaimBuilder[]>`